src/Tests/CaptchaCacheTestCase.php

```diff
@@ -42,7 +42,10 @@ public function testCacheTags() {
     $this->assertEqual($this->drupalGetHeader('x-drupal-cache'), 'HIT');
 
     // Enable captcha on login block and test caching.
-    captcha_set_form_id_setting('user_login_form', 'captcha/Math');
+    /* @var \Drupal\captcha\Entity\CaptchaPoint $captcha_point */
+    $captcha_point = \Drupal::entityManager()->getStorage('captcha_point')->load('user_login_form');
+    $captcha_point->setCaptchaType('captcha/Math');
+    $captcha_point->enable()->save();
```

Comment:
What about making sure that helper function enables() captcha points as well? less changes for this PR.